INSTALL.md

```diff
@@ -21,6 +21,18 @@ If you have added new functions, you will need to add an explicit
 reference to them in
 [docs/study-def-variables.md](./docs/study-def-variables.md).
 
+
+### Updating Cohort Extractor v2 Definitions
+
+cohort-extractor-v2 is a documentation dependency which we can't install directly.
+Instead it generates a JSON file which we commit to this repo.
+
+Run `python -m cohortextractor2 extract-contract-docs` to generate the JSON file.
```

Comment:
thought: Maybe we could later add a GitHub Action in this repository to generate the file, and create a PR if the file changes?

Reply:
I suspect that's the way we'll go, yep.  v2 isn't quite there with its versioning story but I expect we'll do some kind of version check too.